utils.bash

```diff
@@ -0,0 +1,3 @@
+get_pods() {
```

Comment:
@lypht Cool to start, but maybe add a `To-do` note to convert to Golang, since that's our ultimate goal. It is kind of funny though… and a good question… if the work is just parsing `kubectl`, wrapping in Go just to run `sh` may be unnecessary. @rabellamy thoughts?